src/Make.inc

```diff
@@ -35,11 +35,11 @@ else
 endif
 
 LBT_CFLAGS := -g -O2 -std=c99 -fPIC -DLIBRARY_EXPORTS -D_GNU_SOURCE $(CFLAGS)
-LBT_LDFLAGS := $(LDFLAGS)
```

Comment:
Why did you remove the default `LDFLAGS` inheritance?

Reply:
Sorry, that's a mistake (I originally added the flag here so probably I did this when moving it below). https://github.com/JuliaLinearAlgebra/libblastrampoline/pull/53 reverts this.